src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -307,19 +307,24 @@ private IEnumerable<OngoingTask> CollectExternalReplicationTasks(List<ExternalRe
 
         [RavenAction("/databases/*/admin/periodic-backup/config", "GET", AuthorizationStatus.DatabaseAdmin)]
         public async Task GetConfiguration()
+        {
+            await GetBackupConfiguration(this);
+        }
+
+        public static async Task GetBackupConfiguration(RequestHandler requestHandler)
```

Comment:
why not wrap it up in a single processor?

Reply:
done